src/core/crypto/Crypto.ts

```diff
@@ -219,16 +221,16 @@ export class Crypto {
      * @param {string} msg - A text message
      * @param {Uint8Array} iv - An initialization vector
      * @param {Uint8Array} salt - A salt
-     * @param {SignSchema} signSchema The Sign Schema. (KECCAK_REVERSED_KEY / SHA3)
+     * @param {NetworkType} networkType - Catapult network identifier
```

Comment:
I took a slightly different approach.  The crypto component/module knows about SignSchema not about network type. The layer converting from NetworkType to SignSchema is the model layer (Transaction, Addresses, Account, KeyPair, etc) It's more a philosophical comment.

Reply:
Had the same thought as well. First draft didn't touch the signSchema on the core component level. But also consider that signSchema belongs to `NetworkSetting` as Jag mentioned in sig-api. So finally decided to use NetworkType. I think your suggestion makes sense as well. I will change it back.